### ISSUE TITLE
Doesn't work with latest Atom (1.33.0)

### ISSUE THREAD

**Issue by rumax** [AtomLinter/linter-eslint#1208]
## Issue Type
"Bug"

## Issue Description
Doesn't work with latest atom (1.33.0), with 1.30.0 no issues.

## Bug Checklist
- [x] Restart Atom
- [x] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not

Restart/reinstall, etc - nothing works, only downgrade to old Atom (1.30.0).

**Comment by IanVS** [comment]
Hi @rumax I've been using the 1.33 beta for a long time now, and linting works fine for me.  I'm now on 1.34 beta and it also is still working.

Can you do a bit of troubleshooting for us?  Are there any errors showing up in the console?  Are you able to run `Linter Eslint: debug` from the command palette?

**Comment by rumax** [comment]
Hi @IanVS, here it is, I hope it can help:

```
Atom version: 1.33.0
linter-eslint version: 8.4.1
ESLint version: 4.19.1
Hours since last Atom restart: 0
Platform: darwin
Using local project ESLint from: /Users/rumax/projects/PROJECT/node_modules/eslint
Current file's scopes: [
  "source.flow"
]
linter-eslint configuration: {
  "lintHtmlFiles": false,
  "useGlobalEslint": false,
  "showRuleIdInMessage": true,
  "disableWhenNoEslintConfig": true,
  "eslintrcPath": "",
  "globalNodePath": "",
  "advancedLocalNodeModules": "",
  "eslintRulesDirs": [],
  "disableEslintIgnore": false,
  "disableFSCache": false,
  "fixOnSave": false,
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.babel",
    "source.js-semantic"
  ],
  "rulesToSilenceWhileTyping": [],
  "rulesToDisableWhileFixing": [],
  "ignoreFixableRulesWhileTyping": false
}
```

I also tried to use https://atom.io/beta, but same result. The only thing that worked for me is downgrading atom to 1.30.0 and use the plugins I've saved for myself sometime ago.

**Comment by IanVS** [comment]
Ah, yep, you need to add `source.flow` to your list of scopes to lint.

**Comment by rumax** [comment]
Now I see the difference I get after I upgrade atom and dependencies to the latest:

| Atom version: 1.30.0  | Atom version: 1.33.0  |
|---|---|
| "source.js"  | - |
| "punctuation.terminator.statement.js"  | - |
| - | "source.flow" |

Can you, please, tell how to add list of scopes to lint? Is it in config for Atom?

**Comment by IanVS** [comment]
It's a setting in the `linter-eslint` package.

**Comment by rumax** [comment]
Thanks @IanVS , now it works better.

<hr>

This is what I did (_if somebody has the same issue_):

Having default settings:

<img width="599" alt="screen shot 2018-12-05 at 19 51 56" src="https://user-images.githubusercontent.com/2707180/49538558-04a8fa80-f8cc-11e8-9d02-fe818671fa2f.png">

I've added `, source.flow` and restarted Atom. After that everything works again.

The only issue I still see is - if the project has some eslint issues, I've got the notification in the bottom bar only:

<img width="476" alt="screen shot 2018-12-05 at 20 05 39" src="https://user-images.githubusercontent.com/2707180/49538395-95cba180-f8cb-11e8-85ea-99b77b59fa0e.png">

The line with the error is not highlighted:

<img width="230" alt="screen shot 2018-12-05 at 20 05 51" src="https://user-images.githubusercontent.com/2707180/49538415-a3812700-f8cb-11e8-88c5-ccdd0958374f.png">

After fixing the issue and introducing it back, the line highlighting works:

<img width="337" alt="screen shot 2018-12-05 at 20 07 11" src="https://user-images.githubusercontent.com/2707180/49538440-b562ca00-f8cb-11e8-99cc-771e8aaf5a56.png">

<hr>

@IanVS Maybe `list of scopes to lint` should be added to readme? From the other side default settings always worked for me before till recent updates.
